api/envoy/config/metrics/v2/stats.proto

```diff
@@ -200,6 +200,10 @@ message DogStatsdSink {
   }
 
   reserved 2;
+
+  // Optional custom metric name prefix the same as :ref:`StatsdSink's prefix field
```

Comment:
nit: maybe rephrase to something like:

```
// Optional custom metric name prefix. See 
// :ref:`StatsdSink's prefix field <envoy_api_field_config.metrics.v2.StatsdSink.prefix>`
// for more details.

Reply:
Reminder to fix this nit :)

Reply:
Oh, I though I fixed it... I'll fix it now.

Reply:
Thanks, now I fixed this.